proposals/unsafe-evolution.md

```diff
@@ -234,6 +227,16 @@ anonymous and local functions do not keep the unsafe context of their containing
 Is conversion of a lambda or method group marked `unsafe` to a non-unsafe delegate type permitted without warning or error in an `unsafe` context? If we don't do this, then it could be fairly painful
 for various parts of the ecosystem, particularly any enumerables that are passed through LINQ queries.
 
+### Delegate type `unsafe`ty
+
+We could remove the ability to make delegate types as `unsafe` entirely, and simply require that all conversions of `unsafe` lambdas or method groups to a delegate type occur inside an `unsafe` context.
+This could simplify the model around `unsafe` in C#, but at the risk of forcing `unsafe` annotations in the wrong spot and having an area where the real area of `unsafe`ty isn't properly called out.
```

Comment:
If we started without unsafe delegates, can they be added later as an incremental non-breaking change once/if proven necessary?

Reply:
Yes, I expect so.